tests/test_utils/tests.py

```diff
@@ -2145,6 +2090,35 @@ def hook():
         self.assertIsInstance(raised_exception, MyException)
         self.assertEqual(str(raised_exception), "robust callback")
 
+    def test_execute_robust_with_callback_as_partial(self):
```

Comment:
Thanks. Just leaving a note that I had a small look at reducing duplication with the above test via `subTest`, and it wasn't a great fit given the need to pop from `run_on_commit` managed via the `captureOnCommitCallbacks` context manager.